src/handlers/comment/handlers/assign.ts

```diff
@@ -28,7 +22,7 @@ export const assign = async (body: string) => {
     return;
   }
 
-  const opened_prs = await getOpenedPullRequestWithNoReviewsOver24HoursPassedAfterCreated(payload.sender.login);
+  const opened_prs = await getAvailableOpenedPullRequests(payload.sender.login);
 
   logger.info(`Opened Pull Requests with no reviews but over 24 hours have passed: ${JSON.stringify(opened_prs)}`);
```

Comment:
This log comment should be updated because the `24 hours` is configurable.  @wannacfuture